src/collective/zopeconsul/consulserver.py

```diff
@@ -1,7 +1,7 @@
 import logging
 import os
 import re
-from urlparse import urlsplit
+from urllib.parse import urlsplit
```

Comment:
@instification I think we'd want something like this to maintain Python2 compat wouldn't we?
```py
try:
    from urllib.parse import urlparse
except ImportError:
     from urlparse import urlparse
```